R/model_performance.R

```diff
@@ -97,11 +97,15 @@ model_performance <- function(explainer, ..., cutoff = 0.5) {
     fn = sum((observed == 1) * (predicted < cutoff))
 
     measures <- list(
-      recall    = model_performance_recall(tp, fp, tn, fn),
-      precision = model_performance_precision(tp, fp, tn, fn),
-      f1        = model_performance_f1(tp, fp, tn, fn),
-      accuracy  = model_performance_accuracy(tp, fp, tn, fn),
-      auc       = model_performance_auc(predicted, observed)
+      recall            = model_performance_recall(tp, fp, tn, fn),
+      precision         = model_performance_precision(tp, fp, tn, fn),
+      f1                = model_performance_f1(tp, fp, tn, fn),
+      accuracy          = model_performance_accuracy(tp, fp, tn, fn),
+      auc               = model_performance_auc(predicted, observed)
```

Comment:
we lack comma in this line - causing error in test

Reply:
Fixed.